sw/device/silicon_creator/lib/BUILD

```diff
@@ -90,11 +90,23 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "static_critical_epmp_state",
+    srcs = ["static_critical_epmp_state.c"],
+    deps = [
+        ":epmp_state",
+    ],
+    # This library provides a special symbol that the linker will find.
+    # We want this to be a linker input to ensure the weak version in
+    # epmp_state.c is overriden.
+    alwayslink = True,
+)
+
```

Comment:
I think this should be under `sw/device/silicon_creator/lib/base/` along with other static_critical` libs.

Reply:
Done